public/help/misc-preferences.html

```diff
@@ -2,8 +2,8 @@ <h4>Miscellaneous Preferences</h4>
 
 <dl id="help">
   
-  <dt>Turn on Viewing History</dt>
-  <dd>When enabled, viewing history keeps a log of every work you view on the Archive while logged in. You can delete individual works from your History or clear the whole History. If you enable this option and subsequently disable it, works viewed while the option was enabled will still be saved (but you will have to enable the Viewing History again to see them).</dd>
+  <dt>Turn on History</dt>
+  <dd>When enabled, History keeps a log of every work you view on the Archive while logged in. You can delete individual works from your History or clear the whole History. If you enable this option and subsequently disable it, works viewed while the option was enabled will still be saved (but you will have to enable History again to see them).</dd>
```

Comment:
As long as we're here, would you mind changing it from

* "every work you view" to "every work you access"
* "works viewed while the option" to "works accessed while the option"

Reply:
Thank you for taking the time to leave an explanatory comment! I pushed another commit with the changes you detailed. One check failed (failure to reach server), but the same check passed when I ran the test suite on my fork of the code, so I'm not sure if that reflects an issue in the code or a situational network timeout or something of that sort.